apps/web/src/stores/evaluations.ts

```diff
@@ -88,8 +88,8 @@ export default function useEvaluations(
   })
 
   const findEvaluation = useMemo(
-    () => (evaluationUuid: string) =>
-      data.find((e) => e.uuid === evaluationUuid)!,
+    () => (evaluation: EvaluationDto) =>
+      data.find((e) => e.uuid === evaluation.uuid) ?? evaluation,
```

Comment:
Now evaluation will fallback to server evaluation first time `?? evaluation`. But after that `data` will contain a fresh version of it

Reply:
super weird find method which returns its argument instead of null if it doesn't find what it is looking for.  I think you should keep the original implementation and either populate the swr state with the server state at render time or handle the case for findevaludation returning null, which is always a possibility